CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# 4.0.0 (2019-03-11)
+
+### Features
+
+* **rum-core:** add service env to metadata payload ([#198](https://github.com/elastic/apm-agent-rum-js/issues/198)) ([adc038b](https://github.com/elastic/apm-agent-rum-js/commit/adc038b))
+* **rum-core:** Add task API ([#194](https://github.com/elastic/apm-agent-rum-js/issues/194)) ([0153229](https://github.com/elastic/apm-agent-rum-js/commit/0153229))
+* **rum-core:** measure all resource entries in page load ([#173](https://github.com/elastic/apm-agent-rum-js/issues/173)) ([7cd4e0d](https://github.com/elastic/apm-agent-rum-js/commit/7cd4e0d))
+
+
+### Performance Improvements
+
+* **rum-core:** avoid url parsing on resource timing entries ([#174](https://github.com/elastic/apm-agent-rum-js/issues/174)) ([54ea6b9](https://github.com/elastic/apm-agent-rum-js/commit/54ea6b9))
+
+
+### BREAKING CHANGES
```

Comment:
I think the breaking changes should be at the top!

Reply:
This is auto generated, We cannot change the format. Its according to the conventional changelog generator.

Reply:
In almost all changelogs I've seen the breaking changes are at the top of the release notes. see [here](https://github.com/babel/babel/blob/master/CHANGELOG.md#v700-rc4-2018-08-27) for an example.

Didn't you manually copy the change log items here?

Reply:
I did manually, but changelog generator doesn't work that way. Having breaking changes in the bottom on other `packages/changelog.md` and having it in the top on `root changelog` doesnt makes sense.

Reply:
Check lerna changelog - https://github.com/lerna/lerna/blob/master/CHANGELOG.md#300-2018-08-10